.github/workflows/e2e-deploy.yml

```diff
@@ -24,105 +24,49 @@ on:
     runs-on: ubuntu-latest
     timeout-minutes: 60
-    if: startsWith( github.ref, 'refs/tags/') || github.event.action == 'labeled' && github.event.label.name == 'actions/e2e-deploy'
     steps:
       - uses: actions/checkout@v3
-      - name: set git config
+
+      - name: Set Git config
         run: |
           git config --global --add safe.directory ${GITHUB_WORKSPACE}
-      - name: Wait for Docker images
-        if: startsWith( github.ref, 'refs/tags/')
-        id: wait_for_docker_images
-        uses: ./.github/actions/wait-for-docker-image
-      - name: Get PR number
-        id: get_pr_number
-        if: github.event.action == 'labeled' && github.event.label.name == 'actions/e2e-deploy'
-        run: |
-          pr_num=`cat $GITHUB_EVENT_PATH | jq -r ".number"`
-          echo "PR_NUM=${pr_num}" >> $GITHUB_OUTPUT
-      - name: Specify container versions
-        if: github.event.action == 'labeled' && github.event.label.name == 'actions/e2e-deploy'
-        id: specify_container_versions
-        uses: ./.github/actions/detect-docker-image-tags
-        with:
-          tag_name: pr-${{ steps.get_pr_number.outputs.PR_NUM }}
-      - name: Fetch Helm version
-        run: |
-          HELM_VERSION=`make version/helm`
-          echo "helm=${HELM_VERSION}" >> $GITHUB_OUTPUT
-        id: version
-      - uses: rinx/setup-k3d@v0.0.4
-        with:
-          version: latest
-          name: vald
-          agents: 3
-      - name: check k3d
-        run: |
-          kubectl cluster-info
-      - uses: azure/setup-helm@v3
-        with:
-          version: ${{ steps.version.outputs.helm }}
-      - name: Helm version
-        run: |
-          helm version
-      - name: deploy Vald
+
+      - name: Setup E2E environment
+        id: setup_e2e
+        uses: ./.github/actions/setup-e2e
+
+      - name: Deploy Vald
         id: deploy_vald
         uses: ./.github/actions/e2e-deploy-vald
         with:
-          require_libhdf5: "true"
-          helm_extra_options: ${{ steps.specify_container_versions.outputs.HELM_EXTRA_OPTIONS }}
+          helm_extra_options: ${{ steps.setup_e2e.outputs.HELM_EXTRA_OPTIONS }}
           values: .github/helm/values/values-lb.yaml
           wait_for_selector: app=vald-lb-gateway
-      - name: Fetch golang version
-        run: |
-          GO_VERSION=`make version/go`
-          echo "version=${GO_VERSION}" >> $GITHUB_OUTPUT
-        id: golang_version
-      - uses: actions/setup-go@v3
-        with:
-          go-version: ${{ steps.golang_version.outputs.version }}
-      - name: run E2E CRUD
+
+      - name: Run E2E CRUD
         run: |
-          make hack/benchmark/assets/dataset/${DATASET}
-          go version
+          make hack/benchmark/assets/dataset/${{ env.DATASET }}
           make E2E_BIND_PORT=8081 \
-            E2E_DATASET_NAME=${DATASET} \
-            E2E_INSERT_COUNT=10000 \
+            E2E_DATASET_NAME=${{ env.DATASET }} \
+            E2E_INSERT_COUNT=5000 \
```

Comment:
What is a reason for this?

Reply:
sorry, It was changed for verification purposes. I forgot to revert these changes, so I reverted with the following commit :pray:
https://github.com/vdaas/vald/pull/2141/commits/2e81116b67b4f52239f9bd5be8d105eb316fffd8